src/homework5/task2.py

```diff
@@ -0,0 +1,48 @@
+# 2
+# Создайте декоратор, который хранит результаты вызовов функции
+# (за все время вызовов, не только текущий запуск программы)
+
+from datetime import datetime
+
+history = []
+
+
+def history_calls(func):
+    def wrapper(*args, **kwargs):
+        global history
+        result = func(*args, **kwargs)
+        name = func.__name__
```

Comment:
doesn't store results between program calls - contradicts task description. have to use files